spikes/sidecar/README.md

```diff
@@ -0,0 +1,69 @@
+# App Setup
```

Comment:
This is content we deleted when we went to fluent bit but needed for the CPR.  Moved it to spikes to keep for a while in case we need it again